ext/pdo/php_pdo_driver.h

```diff
@@ -308,6 +312,7 @@ struct pdo_dbh_methods {
 	/* if defined to NULL, PDO will use its internal transaction tracking state */
 	pdo_dbh_txn_func		in_transaction;
 	pdo_dbh_get_gc_func		get_gc;
+	pdo_dbh_sql_parser		parser;
```

Comment:
parser -> scanner, here and elsewhere?

Reply:
I had a couple of branches, not 100% sure I pushed the very latest one.

I was a bit undecided between scanner and parser and went back and forth a number of times, but it sounds like scanner is the winner ;-)